### PR TITLE
GUI Doxygen update

### PR DESCRIPTION
**Pull request by couet**
Lot of changes to make GUI oxygen compliant:

- Create the proper groups and put classes in them for GUI 6
- Make the groups structure for ROOT 7
- Fix many typos

The new structure should allow to make a god documentation for ROOT 7. Classes like RFitPanel are now visible in the ref guide.

### CODE REVIEW

**Review by linev — APPROVED** [review]
Huge work Olivier! 
I tried to check all header files - in some of them you auto-repalce `//` by `///<`, making few minor mistakes. But in general everything seems to be ok.

File: gui/ged/inc/TPaveStatsEditor.h [hunk 1]
```diff
@@ -28,23 +21,23 @@ class TPaveStats;
...
-   TGCheckButton      *fSkewness;         // skewness check box
-   TGCheckButton      *fKurtosis;         // kurtosis check box
-   TGCheckButton      *fStatsErrors;      // statistics error check box
-   // widgets for fit options
-   TGCheckButton      *fNameValues;       // parameters' name/values check box
-   TGCheckButton      *fErrors;           // error check box
-   TGCheckButton      *fChisquare;        // Chisquare check box
-   TGCheckButton      *fProbability;      // probability check box
+   TPaveStats         *fPaveStats;        ///< TPaveStats object
+   ///< widgets for stat options
```
Comment (linev) [inline gui/ged/inc/TPaveStatsEditor.h:25]: I have no idea how Doxygen will interpret such comments

File: gui/ged/inc/TPaveStatsEditor.h [hunk 2]
```diff
@@ -28,23 +21,23 @@ class TPaveStats;
...
+   TGCheckButton      *fMean;             ///< mean value check box
+   TGCheckButton      *fRMS;              ///< RMS check box
+   TGCheckButton      *fUnderflow;        ///< underflow number check box
+   TGCheckButton      *fOverflow;         ///< overflow number check box
+   TGCheckButton      *fIntegral;         ///< integral of bins check box
+   TGCheckButton      *fSkewness;         ///< skewness check box
+   TGCheckButton      *fKurtosis;         ///< kurtosis check box
+   TGCheckButton      *fStatsErrors;      ///< statistics error check box
+   ///< widgets for fit options
+   TGCheckButton      *fNameValues;       ///< parameters' name/values check box
```
Comment (linev) [inline gui/ged/inc/TPaveStatsEditor.h:37]: There are very few such cases, may be it is the only file.

**Review by bellenot — APPROVED** [review]
LGTM, thanks Olivier!

File: gui/ged/inc/TPaveStatsEditor.h (hunk 1 above)
Comment (couet) [reply gui/ged/inc/TPaveStatsEditor.h]: I committed the fix. Thanks

File: gui/ged/inc/TPaveStatsEditor.h (hunk 2 above)
Comment (couet) [reply gui/ged/inc/TPaveStatsEditor.h:37]: I will cross check once more.

**Pull request merged**